packages/dendron-next-server/components/graph.tsx

```diff
@@ -0,0 +1,156 @@
+import { createLogger } from "@dendronhq/common-frontend";
+import _ from "lodash";
+import React, { useEffect, useRef, useState } from "react";
+import cytoscape, { Core, ElementsDefinition, EventHandler } from "cytoscape";
+// @ts-ignore
+import euler from "cytoscape-euler";
+import { useRouter } from "next/router";
+import { useThemeSwitcher } from "react-css-theme-switcher";
+import { Space, Typography } from "antd";
+import Head from "next/head";
+
+const getCytoscapeStyle = (themes: any, theme: string | undefined) => `
+      node {
+        width: 5;
+        height: 5;
+        background-color: ${theme === themes.dark ? "#807B7B" : "#999393"};
```

Comment:
instead of hardcoding, can we add this to a constants file  (eg. `styles/theme-antd.tsx`).  in the future, we can use this file to generate the light/dark themes

Reply:
specifically, themes would look like the below

```tsx
type Theme = {
 "background-color": string
...
}

const darkTheme = {...}

const themes = {
  darkTheme,
  ...
}

```

Reply:
Just made these changes - let me know if the structure/design makes sense to you